_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,351 @@
+---
+id: a25f45
+name: Headers attribute specified on a cell refers to cells in the same table element
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute on a cell refer to other cells in the same `table` element.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Jey Nandakumar
+    - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to any `headers` attribute specified on a [`cell`][] within a [`table`][] element, where the [`table`][] element is [visible][] and [included in the accessibility tree][].
+
+## Expectation
+
+Each target attribute is [a set of space separated IDs][], each of which is the ID of an element, that:
+
+1. is a [`cell`][] of the same [`table`][] as the target element.
```

Comment:
Why is this a list of 1? I don't think this should be a list if it has only 1 item.